.clang-tidy

```diff
@@ -1,28 +1,6 @@
 ---
-Checks:          '*'
+Checks:          '*,-fuchsia-*'
 WarningsAsErrors: '*'
-HeaderFilterRegex: '\/src\/'
-AnalyzeTemporaryDtors: false
-CheckOptions:    
-  - key:             google-readability-braces-around-statements.ShortStatementLines
-    value:           '1'
-  - key:             google-readability-function-size.StatementThreshold
-    value:           '800'
-  - key:             google-readability-namespace-comments.ShortNamespaceLines
-    value:           '10'
-  - key:             google-readability-namespace-comments.SpacesBeforeComments
-    value:           '2'
-  - key:             modernize-loop-convert.MaxCopySize
-    value:           '16'
-  - key:             modernize-loop-convert.MinConfidence
-    value:           reasonable
-  - key:             modernize-loop-convert.NamingStyle
-    value:           CamelCase
-  - key:             modernize-pass-by-value.IncludeStyle
-    value:           llvm
-  - key:             modernize-replace-auto-ptr.IncludeStyle
-    value:           llvm
-  - key:             modernize-use-nullptr.NullMacros
-    value:           'NULL'
```

Comment:
Removed this since I don't know its purpose / how it got in there. Thinking it is better to stick with defaults until we have strong opinions on clang-tidy options.

Reply:
Seems fine to add these more intentionally as we go, and probably a good idea to document as a reminder as to why we're adding them. 👍

Reply:
> Removed this since I don't know its purpose / how it got in there. 

My guess, however, is that we created with `clang-tidy -dump-config`.

> Seems fine to add these more intentionally as we go, and probably a good idea to document as a reminder as to why we're adding them. 

Great, glad you agree 👍